README.md

```diff
@@ -102,6 +102,10 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [What is Appwrite and how to contribute for Hacktoberfest](https://dev.to/mishmanners/four-projects-to-contribute-to-this-hacktoberfest-pep)
 * [Appwrite, giving it a shot this Hacktoberfest ‘21](https://medium.com/@anand26shweta/appwrite-giving-it-a-shot-ad1ef79ae44f)
 
+## Hacktoberfest 2022
+* [Appwrite, Hacktoberfest, and how to contribute, including AMA with viewers](https://www.twitch.tv/videos/1634653922)
```

Comment:
Sorry, what part of this video is related to Appwrite?

Reply:
In the middle bit where we featured AppWrite and showed some of the repos to contribute to during Hacktoberfest.

Reply:
Would you please provide the timestamp?

Reply:
I'll find it for you